src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RequesterNameAndAddress.java

```diff
@@ -54,17 +61,21 @@ public String getKey() {
 
     @Override
     public void validate() throws EdifactValidationException {
-        if (identifier.isBlank()) {
+        if (!StringUtils.isBlank(requesterName) && StringUtils.isBlank(identifier)) {
```

Comment:
Can these if statements be grouped in any way to improve readability? `StringUtils.isBlank(identifier)` and `!StringUtils.isBlank(identifier)` have both been repeated a couple of times